### PR TITLE
[EN] Editing practicum captions 08

### PR DESCRIPTION
**Pull request by jemappelleMarie**
You're super cute, and it's so distracting for those who're watching the video of your class ;)
I even think the subtitle auto-generator was distracted as the subtitle was full of mistakes!

### CODE REVIEW

File: docs/en/week08/practicum08.sbv [hunk 1]
```diff
@@ -137,7 +137,7 @@ two macro modules I guess and so the
 input get goes inside an encoder
 
 0:02:09.989,0:02:13.560
-which gives us a code so H which was
+which gives us a code so h which was
```
Comment (telegraphroad) [inline docs/en/week08/practicum08.sbv:140]: 0:01:13.920,0:01:17.900
this network out to layer neural net>>...a two layer...

File: docs/en/week08/practicum08.sbv [hunk 2]
```diff
@@ -173,19 +173,19 @@ as the input
 okay so on the right hand side you have
 
 0:02:44.829,0:02:49.930
-been out encoder on the left hand side
+an auto-encoder, on the left hand side
 
 0:02:47.769,0:02:53.439
 you're gonna be seen what is a variation
```
Comment (telegraphroad) [inline docs/en/week08/practicum08.sbv:179]: variational

File: docs/en/week08/practicum08.sbv [hunk 3]
```diff
@@ -173,19 +173,19 @@ as the input
...
 0:02:49.930,0:02:53.890
-value how to encoder alright so there
+auto-encoder, alright so there
 
 0:02:53.439,0:02:55.900
 you go
 
 0:02:53.890,0:02:57.879
-a variation alt encoder okay it looks
+a variation auto-encoder okay it looks
```
Comment (telegraphroad) [inline docs/en/week08/practicum08.sbv:188]: variational

File: docs/en/week08/practicum08.sbv [hunk 4]
```diff
@@ -206,7 +206,7 @@ the code is actually made of two things
 it's made of one thing that is this kind
 
 0:03:11.230,0:03:18.819
-of capital e of Z and V of Z and they're
+of capital E of z and V of z and they're
```
Comment (telegraphroad) [inline docs/en/week08/practicum08.sbv:209]: 0:03:02.200,0:03:08.439
having the hidden layer age now we have>>hidden layer h

File: docs/en/week08/practicum08.sbv [hunk 5]
```diff
@@ -224,7 +224,7 @@ from this distribution that has been
 parameterize by the encoder and we get
 
 0:03:30.669,0:03:36.790
```
Comment (telegraphroad) [inline docs/en/week08/practicum08.sbv:226]: 0:03:21.189,0:03:27.969
said then we are going to be sampling   >>  said->z

File: docs/en/week08/practicum08.sbv [hunk 6]
```diff
@@ -224,7 +224,7 @@ from this distribution that has been
...
-to Zedd instead it's my latent variable
+to z instead it's my latent variable
```
Comment (telegraphroad) [inline docs/en/week08/practicum08.sbv:227]: z, and z is my latent variable

no "instead" there, might be misleading

File: docs/en/week08/practicum08.sbv [hunk 7]
```diff
@@ -296,19 +296,19 @@ encode the mean and you have basically
 zero variance you're going to get a
 
 0:04:44.940,0:04:52.120
-again a deterministic out encoder so H
+again a deterministic out encoder so h
```
Comment (telegraphroad) [inline docs/en/week08/practicum08.sbv:299]: out encoder>auto-encoder

**Review by telegraphroad — CHANGES_REQUESTED** [review]
Herculean effort, awesome job! Thanks a lot Maryam. I understand how time consuming and tedious it must be to do the captions, especially with the community tools gone.

I went through 5 minutes of practicum 8. I've left some comments. All small stuff and very infrequent, but nevertheless things that could be misleading to people without proper background. It felt like nitpicking but I tried my best not to, only marked the parts that would change the meaning of the sentence. 

Would it be too much to ask a quick review of the transcript, maybe in 2x speed? I did it for mine, just let Yann talk super fast and followed the captions on my file to make sure I haven't missed anything.

I haven't checked the other two captions, might be 100% clean. But anyway it would be wonderful if you could just double-check the first few minutes to see if you've missed anything.

**Review by Atcold — CHANGES_REQUESTED** [review]
Thanks! Can you please fix what @telegraphroad has pointed out?

**Comment by jemappelleMarie** [comment]
Done :)

**Review by telegraphroad — COMMENTED** [review]
Did some spot checks, everything seems to be in order. 
Thanks!

**Review by Atcold — APPROVED** [review]
Awesome!

**Pull request merged**